contracts/source/TestERC20.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.6.3;
+
+contract TestERC20 {
+	event Transfer(address indexed from, address indexed to, uint256 value);
+	event Approval(address indexed owner, address indexed spender, uint256 value);
+	mapping (address => uint256) public balanceOf;
+	mapping (address => mapping (address => uint256)) public allowance;
+	uint256 public totalSupply;
+	bytes public symbol;
+
+	constructor(bytes memory _symbol) public {
+		symbol = _symbol;
```

Comment:
Consider adding `name` too.  It probably won't matter, but it would at least make it technically ERC20.

Reply:
added